### PR TITLE
Add ability to specify HTML font color by name or short hex

### PR DESCRIPTION
**Pull request by boghison**
I think this is useful since it adds a few more options to specify font colors. It would've come in handy when I attempted to set `color="lightblue"`.

### CODE REVIEW

File: fpdf/html.py [hunk 1]
```diff
@@ -21,9 +48,19 @@ def px2mm(px):
...
     if not color:
         return None
-    r = int(color[1:3], 16)
-    g = int(color[3:5], 16)
-    b = int(color[5:7], 16)
+
+    # Checks if color is a name and gets the hex value
+    hexcolor = COLOR_DICT.get(color.lower(), color)
+
+    if len(hexcolor) == 4:
```
Comment (Lucas-C) [inline fpdf/html.py:55]: Why is this "if" needed?

**Review by Lucas-C — CHANGES_REQUESTED** [review]
Nice feature! Thanks :)

* could please add at least one unit test?
* could you add a mention of your addition to the `CHANGELOG.md` file please?
* if you want, you can add your name to the `CONTRIBUTORS.md` file

File: fpdf/html.py (hunk 1 above)
Comment (alexp1917) [reply fpdf/html.py:55]: because html hex codes can also be specified as `#123` as short for `#112233`

**Comment by alexp1917** [comment]
i would like to consider depending on this library instead -- https://pypi.org/project/webcolors/ or something similar but otherwise great idea love it!

**Review by alexp1917 — COMMENTED** [review]
looks good, agree with Lucas's comment about tests, im just a bit hesitant to approve a PR thats like 80% dict literal... not the worst thing in the world, but also not the best.

**Comment by boghison** [comment]
Oh, sure! I didn't really think another dependency is needed for such a small change. Do you want to go with that instead? Because if so, maybe it makes more sense to use it for all color-related stuff.

**Comment by Lucas-C** [comment]
IMHO, I think it'd be nice to merge your PR, once the checks are passing,
and we could do some refactoring later on to introducea dependency, if need be.

**Comment by boghison** [comment]
Sorry, I'll do it in a bit! I will just modify my commit to include all of that since it's not such a big change IMO.

**Comment by mat100payette** [comment]
@Lucas-C Can we expect this to be done by the end of this weekend ? We're very much suffering from the massive pdf sizes and this PR is quite a trivial feature.

**Comment by Lucas-C** [comment]
I published a `v2.4.0` release that does not include this PR

**Comment by boghison** [comment]
@Lucas-C Sorry, did I miss something? I think you need to approve the workflow to test my changes.

**Comment by Lucas-C** [comment]
Approved!

**Comment by boghison** [comment]
@Lucas-C super sorry I didn't realize I had some issues with testing. Should be ok now :)

**Comment by Lucas-C** [comment]
No worries!

Thank you for your contribution @boghison 😉

Merging this now!

**Pull request merged**